### PR TITLE
refactor(logging): blackhole, part 1

### PR DESCRIPTION
**Pull request by 3Hren**

### CODE REVIEW

File: include/cocaine/trace/logger/blackhole.hpp [hunk 1]
```diff
@@ -23,65 +23,63 @@
...
+//     blackhole::record_t
+//     open_record(Level level, blackhole::attribute::set_t attributes = blackhole::attribute::set_t()) const {
+//         // TODO: Do this under lock or drop assignment.
+//         AttributeFetcher fetcher;
+//         const auto& dynamic_attributes = fetcher();
+//         std::copy(this->attributes.begin(), this->attributes.end(), std::back_inserter(attributes));
+//         std::copy(dynamic_attributes.begin(), dynamic_attributes.end(), std::back_inserter(attributes));
+//         return this->wrapped->open_record(level, std::move(attributes));
+//     }
+// };
```
Comment (3Hren) [inline include/cocaine/trace/logger/blackhole.hpp]: We need to restore this. This code just checks if there is trace id attached and adds it (with span and parent) to the current attributes set.

Am I right?

Comment (3Hren) [inline include/cocaine/trace/logger/blackhole.hpp]: @antmat ^

File: src/service/logging.cpp [hunk 2]
```diff
@@ -27,34 +27,35 @@
...
+    //             context.config.logging.loggers.at(backend).verbosity
+    //         ));
+    //     } catch(const std::out_of_range& e) {
+    //         throw cocaine::error_t("logger '%s' is not configured", backend);
+    //     }
+    //
+    //     wrapper.reset(new log_t(*logger, {{"source", cocaine::format("%s/%s", name, backend)}}));
+    // } else {
+    //     wrapper = context.log(cocaine::format("%s/core", name));
+    // }
```
Comment (3Hren) [inline src/service/logging.cpp]: Mark this not to forget.

File: src/storage/files.cpp [hunk 3]
```diff
@@ -55,10 +59,10 @@ files_t::read(const std::string& collection, const std::string& key) {
         );
     }
 
-    COCAINE_LOG_DEBUG(m_log, "reading object '%s'", key)(
-        "collection", collection,
-        "path", file_path
-    );
+    COCAINE_LOG_DEBUG(m_log, "reading object '{}'", key, attribute_list({
+        {"collection", collection}
+        // {"path", file_path}
```
Comment (3Hren) [inline src/storage/files.cpp]: Think about how to implement old `operator<<` behavior for non-generic attributes.

File: include/cocaine/trace/logger/blackhole.hpp (hunk 1 above)
Comment (antmat) [inline include/cocaine/trace/logger/blackhole.hpp]: Yep.

**Comment by 3Hren** [comment]
Updated.

To do list:
- [ ] Fix tracing.
- [ ] Fix non-core logging services.
- [ ] Fix custom attributes in files storage.

**Pull request closed without merging**